### PR TITLE
test/pkg_u8g2: increase main stacksize for SDL

### PR DESCRIPTION
**Pull request by kfessel**
### Contribution description

thanks @fjmolinas #16944

I found a partial solution to #13501 #16714

and added that information to pkg_u8g2 doc.txt

### Testing procedure

```
RIOT/tests/pkg_u8g2$ BOARD=native TEST_OUTPUT=2 make term
```
segfault before patch does not segfault after patch

### Issues/PRs references

fixes the segfault part of #13501 #16714

### CODE REVIEW

File: tests/pkg_u8g2/Makefile [hunk 1]
```diff
@@ -30,6 +30,8 @@ ifeq ($(TEST_OUTPUT),1)
 endif
 
 ifeq ($(TEST_OUTPUT),2)
+# SDL requires more stack
+  CFLAGS += -DTHREAD_STACKSIZE_MAIN=64*1024
```
Comment (fjmolinas) [inline tests/pkg_u8g2/Makefile:34]: I don't remember if I used 64Kb just cause or if I played around with the value a bit, did you try it out?

Comment (kfessel) [reply tests/pkg_u8g2/Makefile:34]: done 32k wasn't enough

File: pkg/u8g2/doc.txt [hunk 2]
```diff
@@ -3,4 +3,14 @@
  * @ingroup  pkg
  * @brief    Provides a monochrome graphics library for OLED and LCD displays
  * @see      https://github.com/olikraus/u8g2
- */
\ No newline at end of file
+ *
+ * ## Requirements
+ *
+ * using SDL requires more stack so in case you are using it add
```
[the remark this answers is not in the document]
Comment (kfessel) [reply pkg/u8g2/doc.txt:9]: done

File: tests/pkg_u8g2/Makefile (hunk 1 above)
Comment (kfessel) [reply tests/pkg_u8g2/Makefile]: https://github.com/rofl0r/df-libgraphics/blob/nogl/dist/sdl-stacksize.patch < dwarf-fortress has a 256kB patch for musl+sdl  (but they also have much more on display)

Comment (fjmolinas) [reply tests/pkg_u8g2/Makefile]: Lets leave the minimum that works in the Makefile, but how about adding a hint that more could be required in the doc.txt?

**Review by fjmolinas — APPROVED** [review]
ACK!

**Pull request merged**